src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -33,7 +36,10 @@ export class TemplateEditorComponent extends UnsubscribeOnDestroyAdapter impleme
 
   templateEditor: FormGroup;
 
-  DATAFIELDS: any = (dataFields as any).default;
+  DATAFIELDSSTUDENT: any = (dataFieldsStudent as any).default;
+  DATAFIELDSCOURSE: any = (dataFieldsCourse as any).default;
+  DATAFIELDSSTANDARD: any = (dataFieldsStandardAcc as any).default;
+  DATAFIELDSNURSING: any = (dataFieldsNursingAcc as any).default;
```

Comment:
It would be good to have type information for those. Types can be generated with tools such as http://json2ts.com/

Reply:
Typed the data field as suggested.